doc/nest_by_example/NEST by Example - An Introduction to the Neural Simulation Tool NEST Version 2.12.0.ipynb

```diff
@@ -0,0 +1,2437 @@
+    "that are not set explicitly are initialized with default values. You\n",
+    "can display them with\n",
+    "`nest.GetDefaults(model_name)`.\n",
+    "Note that only the first\n",
+    "parameter of `Create` is mandatory. \n",
+    "\n",
+    "`Create` returns a list of integers, the  global\n",
+    "identifiers (or GID for short) of each node created. The GIDs are\n",
+    "assigned in the order in which nodes are created. The first node is\n",
+    "assigned GID 1, the second node GID 2, and so on.\n",
+    "\n",
+    "After creating the neuron, sine and noise generators, and voltmeter, we connect the nodes. First we connect the sine generator and the voltmeter\n",
+    "to the neuron. The command `Connect` takes two or more\n",
+    "arguments. The first argument is a list of source nodes. The second\n",
+    "argument is a list of target nodes. `Connect` iterates these\n",
+    "two lists and connects the corresponding pairs.\n",
+    "\n",
+    "A node appears in the source position of `Connect` if it sends events\n",
+    "to the target node. In our example, the sine generator is in the\n",
+    "source position because it injects an alternating current into the\n",
+    "neuron. The voltmeter is in the source position, because it polls the\n",
+    "membrane potential of the neuron. Other devices may be in the target\n",
+    "position, e.g., the spike detector which receives spike events from a\n",
+    "neuron. If in doubt about the order, consult the documentation of the\n",
+    "respective nodes using NEST's help system. For example, to read the\n",
+    "documentation of the ac\\_generator you can type\n",
+    "`nest.help('ac_generator')`. \n",
+    "Dismiss the help by typing `q`.\n",
+    "\n",
+    "Next, we use the command `Connect` with the\n",
+    "`syn_spec` parameter to connect the\n",
+    "two Poisson generators to the neuron. In this example the synapse\n",
+    "specification `syn_spec` provides only weight and delay\n",
+    "values, in this case $\\pm 1$ pA input current amplitude and $1$ ms\n",
+    "delay. We will see more advanced uses of  `syn_spec` below.\n",
+    "\n",
+    "After connecting the nodes, the network is ready. The following\n",
+    "line calls the NEST function `Simulate` which runs the\n",
+    "network for 1000 milliseconds. The function returns after the\n",
+    "simulation is finished. Then, function `voltage_trace` is\n",
+    "called to plot the membrane potential of the neuron. If you are\n",
+    "running the script for the first time, you may have to tell Python to display\n",
+    "the figure by typing `pylab.show()`.\n",
+    "\n",
+    "If you want to inspect how your network looks so far, you can print\n",
+    "it using the command `PrintNetwork()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "nest.PrintNetwork(2)"
```

Comment:
My bad, should have read what was written below a little more carefully...
Ok! Everything is working under py3, then, I'll start reading the text ASAP ;)